Octokit.Reactive/Clients/IObservableAuthorizationsClient.cs

```diff
@@ -19,6 +19,19 @@ public interface IObservableAuthorizationsClient
         IObservable<Authorization> GetAll();
 
         /// <summary>
+        /// Get all <see cref="Authorization"/>s for the authenticated user. This method requires basic auth.
+        /// </summary>
+        /// <remarks>
+        /// See <a href="http://developer.github.com/v3/oauth/#list-your-authorizations">API documentation</a> for more
+        /// details.
+        /// </remarks>
+        /// <param name="options">Options for changing the API response</param>
+        /// <returns>An <see cref="Authorization"/></returns>
```

Comment:
I'm not sure why but in the rest of the lib the `<returns>` are left empty.
We should either stick with that - or start here as change (which would necessitate going back and changing the entire code base)